deploy/kubernetes/Jenkinsfile.yml

```diff
@@ -18,5 +18,5 @@ stages:
             make check-no-changes;
         stage: checks
     k8sTest:
-        k8sTest: "v1.29.0,v1.28.0,v1.27.3,v1.26.6"
+        k8sTest: "v1.30.0,v1.29.0,v1.28.0,v1.27.3"
```

Comment:
Can we update the list based on the https://kubernetes.io/releases/?

Reply:
As kind is used to spin up the kubernetes clusters, we can only use the available kindest/node as found in https://hub.docker.com/r/kindest/node/tags?page=&page_size=&ordering=&name=.

The latest for 1.27 available is v1.27.13
For 1.28 the latest available  is v1.28.9
For 1.29 the latest available is v1.29.4.

Do you suggest to use those or keep the ones we already have?

Reply:
The latter the better then. So I would say to update them